src/sections/disease/OTProjects/index.js

```diff
@@ -0,0 +1,10 @@
+export const definition = {
+  id: 'otarProjects',
+  name: 'Open Targets Projects',
+  shortName: 'OP',
+  // TODO: determine if there's data using real data from API
+  hasData: () => true,
```

Comment:
So for the moment we use the data from the json, and will update to API at a later stage?

Reply:
@LucaFumis yes, that's right

Reply:
Great! thanks @mirandaio